autonomy/cli/analyse.py

```diff
@@ -165,37 +168,47 @@ def parse_logs(file: Path) -> None:
         raise click.ClickException(str(e)) from e
 
 
-@abci_group.command(name="check-handlers")
-@click.argument(
-    "packages_dir",
-    type=click.Path(dir_okay=True, exists=True),
-    default=Path.cwd() / "packages",
-)
+@analyse_group.command(name="handlers")
+@pass_ctx
 @click.option(
-    "--skip",
+    "--common-handlers",
+    "-h",
     type=str,
-    default="abstract_abci",
-    help="Specify which skills to skip. Eg. skill_0,skill_1,skill_2",
+    default=[
+        "abci",
```

Comment:
Why is this default?

Reply:
On an assumption that ABCI handler is going to be present in every skill, should we remove the defaults?

Reply:
looks like a mutable default argument to me :grimacing: